### DISCUSSION TITLE
Tickbox to Enable or Disable environmental variables?

### DISCUSSION THREAD

**Discussion by sallymanda** [lutris/lutris#5152] [Ideas]
Hey everyone :-) I would like to have a little tickbox next to each environmental variable so I can enable or disable the variable without having to delete it entirely. It's a minor feature, but would certainly assist with troubleshooting games that are on the trickier side to get running properly, and even for some basic things like enabling and disabling HUDs, etc.

**Comment by TriMoon** [comment]
Sorry to wake a dead topic, but...
You don't need to delete the variable at all, just put a dot infront of the environment variable's name...
That way it won't be recognized anymore and you still have an easy way to undo it...
